### PR TITLE
Allow swagger on disabled versions.

### PR DESCRIPTION
**Pull request by fisx**
https://wearezeta.atlassian.net/browse/SQPIT-1691

## Checklist

 - [x] Add a new entry in an appropriate subdirectory of `changelog.d`
 - [x] Read and follow the [PR guidelines](https://docs.wire.com/developer/developer/pr-guidelines.html)

### CODE REVIEW

**Comment by fisx** [comment]
feel free to merge right away if you approve.

File: libs/wire-api/src/Wire/API/Routes/Version/Wai.hs [hunk 1]
```diff
@@ -61,6 +61,13 @@ parseVersion req = do
 looksLikeVersion :: Text -> Bool
 looksLikeVersion version = case T.splitAt 1 version of (h, t) -> h == "v" && T.all isDigit t
 
+-- | swagger-delivering end-points are not disableable: they should work for all versions.
+requestIsDisableable :: Request -> Bool
+requestIsDisableable (pathInfo -> path) = case path of
+  ["api", "swagger-ui"] -> False
```
Comment (smatting) [inline libs/wire-api/src/Wire/API/Routes/Version/Wai.hs:67]: It think `api-internal` is missing. See e.g. https://staging-nginz-https.zinfra.io/api-internal/swagger-ui/brig

**Review by smatting — CHANGES_REQUESTED** [review]

File: libs/wire-api/src/Wire/API/Routes/Version/Wai.hs (hunk 1 above)
Comment (smatting) [reply libs/wire-api/src/Wire/API/Routes/Version/Wai.hs:67]: Err nevermind, internal apis are not versioned :)

**Review by smatting — APPROVED** [review]

File: libs/wire-api/src/Wire/API/Routes/Version/Wai.hs (hunk 1 above)
Comment (fisx) [reply libs/wire-api/src/Wire/API/Routes/Version/Wai.hs:67]: no, wait.  didn't we want to make internal apis non-versioned?

**Pull request merged**